### PR TITLE
989 Fix

### PR DESCRIPTION
**Pull request by amber-emmes**
vialMapping in kit CSV now has home collection values for all sites, plus a default value. (989)

### CODE REVIEW

File: src/pages/homeCollection/kitCSV.js [hunk 1]
```diff
@@ -36,7 +36,9 @@ const csvFileButtonSubmit = () => {
       showAnimation();
       try {
           const results = await getKitsByReceivedDate(dateString);
+          console.log('results', results);
```
Comment (JoeArmani) [inline src/pages/homeCollection/kitCSV.js:39]: please remove the console log if we don't need it

File: src/pages/homeCollection/kitCSV.js [hunk 2]
```diff
@@ -36,7 +36,9 @@ const csvFileButtonSubmit = () => {
...
           const modifiedResults = modifyKitQueryResults(results.data);
+          console.log('modifiedResults', modifiedResults);
```
Comment (JoeArmani) [inline src/pages/homeCollection/kitCSV.js:41]: same here

File: src/pages/homeCollection/kitCSV.js [hunk 3]
```diff
@@ -111,15 +113,36 @@ const modifyKitQueryResults = (kitsData) => {
 
 const getVialTypesMapping = (collectionType, site, tubeId) => {
   if (collectionType === 'home') {
-    return vialMapping[collectionType]?.[site]?.[tubeId] || ['', '', '', ''];
+    return vialMapping[collectionType]?.[site]?.[tubeId] || ["15ml Nalgene jar",	"Crest Alcohol Free",	"Saliva",	"10"];
```
Comment (JoeArmani) [inline src/pages/homeCollection/kitCSV.js:116]: Looks like all the values are the same, so you may be able to just return the fallback value
`["15ml Nalgene jar", "Crest Alcohol Free", "Saliva", "10"]`
and remove the vialMapping object.

**Review by JoeArmani — APPROVED** [review]
looks good, minor comments

File: src/pages/homeCollection/kitCSV.js (hunk 3 above)
Comment (amber-emmes) [reply src/pages/homeCollection/kitCSV.js]: Done

**Review by jhflorey — APPROVED** [review]
Look good to me! approve

**Pull request merged**